packages/x-components/src/components/result/base-result-previous-price.vue

```diff
@@ -9,7 +9,7 @@
          @binding {result} result - Result data
     -->
     <slot :result="result">
-      <BaseCurrency :value="result.price.originalValue" :format="format" />
+      <BaseCurrency v-bind="{ value: result.price.originalValue, ...(format && { format }) }" />
```

Comment:
Due to how `propsData` works I can't send anything as `format` prop if I don't want it to appear there.